tests/management/Makefile

```diff
@@ -10,8 +10,11 @@ PROJECT=gongyuan-pipeline-test
 MANAGED_PROJECT=gongyuan-pipeline-test
 BUCKET=gongyuan-test-5
 
+# Test master
+MANAGEMENT_URL=https://github.com/kubeflow/gcp-blueprints.git/management@master
+
 # Test 1.1.0
-MANAGEMENT_URL=https://github.com/kubeflow/gcp-blueprints.git/management@v1.1.0
+# MANAGEMENT_URL=https://github.com/kubeflow/gcp-blueprints.git/management@1.1.0
```

Comment:
Give this a separate name, like `MANAGEMENT_1_1_0_URL` ? May need to parameterize the URL.

Reply:
If we uncomment this, then we can test 1.1.0, so it has to be the same name

Reply:
Understand, just a low priority thinking that if we can parameterize this as if it were an input to a function, then we can do `make test-master` or `make test-kf-1-1`, and it will pick up the right URI without the need that someone manually comments/uncomments the code.